addons/hr_expense/static/src/views/attachment_view.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.AttachmentView" t-inherit-mode="extension">
+        <xpath expr="//h3/div/t" position="before">
+            <span t-if="state.thread.model==='hr.expense.sheet'">
```

Comment:
What is the purpose of this span? I believe that you could put the condition directly on the inner `t`.
By the way, it is common practice in JavaScript to put spaces around the `===` operator